packages/msgraph/test/Adaptor.test.js

```diff
@@ -356,7 +355,7 @@ describe('getFile', () => {
     };
 
     const finalState = await execute(
-      getFile('01LUM6XOGRONYNTZ26DBBJPTN5IFTQPBIW')
+      getFile('01LUM6XOGRONYNTZ26DBBJPTN5IFTQPBIW', { parseAs: 'text' })
```

Comment:
This should not be passing `parseText`. I want the unit tests to be unaffected by this change.